src/Http/WebUtilities/src/HttpRequestStreamReader.cs

```diff
@@ -167,11 +167,6 @@ public override int Read(char[] buffer, int index, int count)
     /// <inheritdoc />
     public override int Read(Span<char> buffer)
     {
-        if (buffer == null)
-        {
-            throw new ArgumentNullException(nameof(buffer));
-        }
-
```

Comment:
Span can't be null. I looked at other `TextReader` implementations and they all no-op and return a read count of 0. Removed this check.